src/Store/Reducers/SystemCvesStore.js

```diff
@@ -56,33 +53,31 @@ function fulfilledVulnerabilities(state, action) {
                 payload,
                 isLoading: false
             },
-            prevLoadedRows: unionBy(action.payload.data, state.prevLoadedRows, 'id'),
             ...state.isAllExpanded && { expandedRows: action.payload.data.map(({ id }) => ({ id, isOpen: true })) }
         };
     }
 
     return state;
 }
 
-// #TODO refactor to use selectRows helper
 function selectEntity(state, action) {
```

Comment:
I know you have not changed this in this PR, but can we please rename this to `selectCve`? We use word `entity` to refer to system.